qiime2/core/testing/method.py

```diff
@@ -49,6 +49,11 @@ def long_description_method(mapping1: dict, name: str, age: int) -> dict:
     return {name: age}
 
 
+def docstring_order_method(req_input: dict, req_param: str,
+                           opt_input: dict=None, opt_param: int=None) -> dict:
+    return{req_param: opt_param}
```

Comment:
style request: can you put a space between `return` and the `dict`?

Reply:
Done. good catch, thanks!